layers/+lang/common-lisp/packages.el

```diff
@@ -15,7 +15,16 @@
         ggtags
         helm
         helm-gtags
-        slime))
+        slime
+        auto-completion
+        (company :toogle lisp-mode-enable-slime-company)
```

Comment:
You have no company-specific configuration, so you don't need to list company in the packages list. `lisp-mode-enable-slime-company` should be part of slime-company's toggle.